### ISSUE TITLE
Installation InstaPy W10:  selenium.common.exceptions.WebDriverException: Message: TypeError: browsingContextFn().currentWindowGlobal is null

### ISSUE THREAD

**Issue by gianzzzzzz** [InstaPy/InstaPy#6017]
## InstaPy configuration

hi after i solve about 5/6 different error thanks to the comment on youtube i can't succed in solving this one:
selenium.common.exceptions.WebDriverException: Message: TypeError: browsingContextFn().currentWindowGlobal is null

i'm on w10, i installed all the programs today, i should have all the latest version

do you know what should i do? thanks in advance

**Comment by Tr1pke** [comment]
you haven't solved the biggest error of them all, using windows ...

**Comment by gianzzzzzz** [comment]
Stop bullying :(

**Comment by Cph4v** [comment]
yes i have this problem when i try to login one of my accounts please answere...i'm in  windows10
and i have this problem just for one of my accounts not for all...

**Comment by elulcao** [comment]
Could you check if the geckodriver is well defined for the InstaPy session.
There is an example in the official documentation
[InstaPy](https://instapy.org)

**Comment by windowshopr** [comment]
Go to line 1953 of ".../instapy/util.py" and change "PFL" to "VOEL". Worked for me on Windows 10, python 3.6.

EDIT:

Change line

explicit_wait(browser, "PFL", [], logger, 10)

to 

explicit_wait(browser, "VOEL", [], logger, 10)

**Comment by Cph4v** [comment]
tnx...i update my geckodriver and it works

**Comment by Cph4v** [comment]
tnx...i update my geckodriver and it worked..

**Comment by shyland89** [comment]
> Go to line 1953 of ".../instapy/util.py" and change "PFL" to "VOEL". Worked for me on Windows 10, python 3.6.
> 
> EDIT:
> 
> Change line
> 
> explicit_wait(browser, "PFL", [], logger, 10)
> 
> to
> 
> explicit_wait(browser, "VOEL", [], logger, 10)

Hi there, how do I go about changing this on a Mac, running into the same login issue currently. Thanks!
